models/get-reviews-id-models.js

```diff
@@ -0,0 +1,15 @@
+const db = require('../db/connection');
+const reviews = require('../db/data/test-data/reviews');
+
+function fetchReviewsByID(reviewId){
+
+    return db.query(`SELECT * FROM reviews WHERE review_id = $1`, [reviewId])
+    .then((result) => {
+        if(result.rowCount === 0){
+            return Project.reject({status: 404, msg: 'End-point Not Found'});
```

Comment:
The error message here isn't correct. The endpoint does exist; the review_id is just not in the database. I would rewrite this to make it more distinct from the 'invalid endpoint' 404 message